models/trucks/partner_truck_query.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+// PartnerTrucksQuery is the struct for making dynamodb queries for partner trucks
+type PartnerTrucksQuery struct {
```

Comment:
Move this struct to other file and you can do part of the logic you do in the lambda here, for consistency with the rest of the models